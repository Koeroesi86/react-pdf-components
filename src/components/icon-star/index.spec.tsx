import React from "react";
import { renderToString } from "@react-pdf/renderer";
import { Wrapper } from "@test-helpers";
import IconStar from "./index";

describe("<IconStar/>", () => {
  it("should render", async () => {
    const result = await renderToString(
      <Wrapper>
        <IconStar width={12} height={12} color="#000"/>
      </Wrapper>
    );
    expect(result).toMatchSnapshot();
  });
});
