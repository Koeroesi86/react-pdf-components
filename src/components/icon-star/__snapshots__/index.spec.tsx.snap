// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconStar/> should render 1`] = `
"%PDF-1.3
%����
7 0 obj
<<
/Type /Page
/Parent 1 0 R
/MediaBox [0 0 595.280029 841.890015]
/Contents 5 0 R
/Resources 6 0 R
>>
endobj
6 0 obj
<<
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI]
>>
endobj
5 0 obj
<<
/Length 734
>>
stream
1 0 0 -1 0 841.890015 cm
q
q
1 0 0 1 0 0 cm
q
q
0 3 m
12 3 l
12 3 12 3 12 3 c
12 15 l
12 15 12 15 12 15 c
0 15 l
0 15 0 15 0 15 c
0 3 l
0 3 0 3 0 3 c
h
W n
1 0 0 1 0 3 cm
0.020833 0 0 0.020833 0 0 cm
1 0 0 1 0 32 cm
q
/DeviceRGB cs
0 0 0 scn
528.1 171.5 m
382 150.2 l
316.7 17.8 l
305 -5.8 271.1 -6.1 259.3 17.8 c
194 150.2 l
47.9 171.5 l
21.7 175.3 11.2 207.6 30.2 226.1 c
135.9 329.1 l
110.9 474.6 l
106.4 500.9 134.1 520.6 157.3 508.3 c
288 439.6 l
418.7 508.3 l
441.9 520.5 469.6 500.9 465.1 474.6 c
440.1 329.1 l
545.8 226.1 l
564.8 207.6 554.3 175.3 528.1 171.5 c
h
388.6 312.3 m
412.3 450.7 l
288 385.4 l
163.7 450.7 l
187.4 312.3 l
86.8 214.3 l
225.8 194.1 l
288 68.1 l
350.2 194.1 l
489.2 214.3 l
388.6 312.3 l
h
f
Q
Q
Q
Q
Q

endstream
endobj
9 0 obj
(react-pdf)
endobj
10 0 obj
(react-pdf)
endobj
11 0 obj
(D:20210414183756Z)
endobj
8 0 obj
<<
/Producer 9 0 R
/Creator 10 0 R
/CreationDate 11 0 R
>>
endobj
3 0 obj
<<
/Type /Catalog
/Pages 1 0 R
/Names 2 0 R
/ViewerPreferences 4 0 R
>>
endobj
1 0 obj
<<
/Type /Pages
/Count 1
/Kids [7 0 R]
>>
endobj
2 0 obj
<<
/Dests <<
  /Names [
]
>>
>>
endobj
4 0 obj
<<
/DisplayDocTitle true
>>
endobj
xref
0 12
0000000000 65535 f 
0000001237 00000 n 
0000001294 00000 n 
0000001150 00000 n 
0000001341 00000 n 
0000000200 00000 n 
0000000133 00000 n 
0000000015 00000 n 
0000001076 00000 n 
0000000985 00000 n 
0000001012 00000 n 
0000001040 00000 n 
trailer
<<
/Size 12
/Root 3 0 R
/Info 8 0 R
>>
startxref
1384
%%EOF
"
`;
